From 181d3715247d6e1ca800147ddb5592fdfdd94de0 Mon Sep 17 00:00:00 2001
From: Richard Purdie <richard@openedhand.com>
Date: Tue, 30 Oct 2007 08:21:24 +0000
Subject: [PATCH] perl: Sync with OE, fix hardcoded staging references

Upstream-Status:Inappropriate [embedded specific]

---
 installperl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/installperl b/installperl
index af61203..f12441d 100755
--- a/installperl
+++ b/installperl
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir '..' if !-d 'lib' and -d '../lib';
-    @INC = 'lib';
-    $ENV{PERL5LIB} = 'lib';
+#    @INC = 'lib';
+#    $ENV{PERL5LIB} = 'lib';
 
     # This needs to be at BEGIN time, before any use of Config
     # install_lib itself loads and imports Config into main::
